### ISSUE TITLE
Documentation compilation

### ISSUE THREAD

**Issue by CSDUMMI** [orbitdb/welcome#96]
Currently, the Documentation of OrbitDB is hosted on GitHub
in the various Repositories.

But this makes them very hard to search and reference quickly.

I propose, that we compile all of these documentation
files into a single easy to reference repository, that can then be put
up in a cook book.

Either by hand once or via a script.

TODO:
- [x] Copy API.md and GUIDE.md
- [ ]  Merge `orbit-db` README with API.md
- [ ]  Merge `orbit-db-store` README with API.md 
- [ ] Merge `orbit-db-feedstore` README with API.md
- [ ]  Merge `orbit-db-eventstore`
- [ ]  Merge `orbit-db-docstore`
- [ ]  Merge `orbit-db-kvstore`
- [ ]  Merge `orbit-db-counterstore`
- [ ]  Merge `crdts`
- [ ] Merge `orbit-db-pubsub`
- [ ]  Merge `orbit-db-cache`
- [ ]  Merge `orbit-db-identity-provider`
- [ ]  Merge `orbit-db-keystore`
- [ ]  Merge `orbit-db-io`
- [ ]  Merge `ipfs-log`

**Comment by CSDUMMI** [comment]
After we made this compilation, the documentation in the repositories READMEs are discontinued
and prefaced with a link to the relevant pages in the Documentations Repository/Page.

**Comment by aphelionz** [comment]
I think we should make a new Section of the Field Manual called "Cookbook" after the Tutorial https://github.com/orbitdb/field-manual

**Comment by CSDUMMI** [comment]
That would be a position, that people would expect it to be, wouldn't they?

Is the Field Manual already compiled into a web page?

If so, this would be the way that I would want this to go.

**Comment by CSDUMMI** [comment]
But I also want to move the different README Files of all the Repos into the API.md file.

**Comment by CSDUMMI** [comment]
> Currently, the Documentation of OrbitDB is hosted on GitHub
> in the various Repositories.
> 
> But this makes them very hard to search and reference quickly.
> 
> I propose, that we compile all of these documentation
> files into a single easy to reference repository, that can then be put
> up in a cook book.
> 
> Either by hand once or via a script.
> 
> TODO:
> 
> * [x]  Copy API.md and GUIDE.md
> * [ ]   Merge `orbit-db` README with API.md
> * [ ]   Merge `orbit-db-store` README with API.md
> * [ ]  Merge `orbit-db-feedstore` README with API.md
> * [ ]   Merge `orbit-db-eventstore`
> * [ ]   Merge `orbit-db-docstore`
> * [ ]   Merge `orbit-db-kvstore`
> * [ ]   Merge `orbit-db-counterstore`
> * [ ]   Merge `crdts`
> * [ ]  Merge `orbit-db-pubsub`
> * [ ]   Merge `orbit-db-cache`
> * [ ]   Merge `orbit-db-identity-provider`
> * [ ]   Merge `orbit-db-keystore`
> * [ ]   Merge `orbit-db-io`
> * [ ]   Merge `ipfs-log`

**Comment by aphelionz** [comment]
> Is the Field Manual already compiled into a web page?

No, but it could be. Moving everything to API.md would be a good step in the right direction too.
